main.py

```diff
@@ -131,6 +136,43 @@ def _evaluate(pipeline_name, dev_mode):
     ctx.channel_send('Recall', 0, recall)
 
 
+def _evaluate_in_chunks(pipeline_name, dev_mode, chunk_size):
+    meta = pd.read_csv(os.path.join(params.meta_dir, 'stage{}_metadata.csv'.format(params.competition_stage)))
+    meta_valid = meta[meta['is_valid'] == 1]
+
+    if dev_mode:
+        meta_valid = meta_valid.sample(30, random_state=1234)
+
+    logger.info('processing metadata of shape {}'.format(meta_valid.shape))
+
+    evaluation_chunks = []
+    for meta_chunk in generate_data_frame_chunks(meta_valid, chunk_size):
+        data = {'input': {'meta': meta_chunk,
+                          'meta_valid': None,
+                          'train_mode': False,
+                          'target_sizes': [(300, 300)] * len(meta_chunk)
+                          },
+                }
+
+        pipeline = PIPELINES[pipeline_name]['inference'](SOLUTION_CONFIG)
+        pipeline.clean_cache()
+        output = pipeline.transform(data)
+        pipeline.clean_cache()
+        y_pred = output['y_pred']
+
+        evaluation_chunk = create_submission(meta_chunk, y_pred, logger, CATEGORY_IDS)
+        evaluation_chunks.extend(evaluation_chunk)
```

Comment:
@taraspiotr It seems that `evalution_chunks` and `prediction_chunks` name no longer makes sense since it is actually predicted_annotations list that is extended with every chunk.